tests/e2e/install_claimed_browser.py

```
@@ -0,0 +1,470 @@
+        ]
+    )
+    run(["sudo", "apt-get", "update"])
+    run(["sudo", "apt-get", "install", "-y", "brave-browser"])
+
+
+def install_opera_apt() -> None:
+    run(["sudo", "apt-get", "install", "-y", "curl", "gnupg"])
+    key = Path("/usr/share/keyrings/opera-browser.gpg")
+    with urllib.request.urlopen("https://deb.opera.com/archive.key") as response:
+        payload = response.read()
+    subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)